style.css

```diff
@@ -100,3 +100,273 @@ section {
   width: 170.17px;
   height: 20px;
 }
+
+/* ------------------------------CARDS---------------------------------- */
+
+/* Card-container general settings */
+
+/* note: I've placed the images in the html */
+
+.card-container {
+  display: grid;
+  background-color: white;
+  border: 1px solid #dfe1e6;
+  grid-template-columns: auto;
+  margin: 5%;
+  padding: 5%;
+  border-radius: 1em;
+  text-align: center;
+}
+
+/* general settings for every Cards */
+.cardWorks {
+  text-align: start;
+}
+
+/* Editing the TONIC word inside the card */
+div > h4 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: rgba(23, 43, 77, 1);
+}
+
+/* editing "Back End Dev", "2015" and dots in the card */
+.dot {
+  height: 8px;
+  width: 8px;
+  background-color: rgba(193, 199, 208, 1);
+  border-radius: 50%;
+  display: inline-block;
+  margin-left: 8px;
+}
+
+/* Editing the word: "CANOPY" */
+div .word1 {
+  color: rgba(52, 69, 99, 1);
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 19px;
+}
+
+/* Editing the word: "Back End Dev" */
+div .word2 {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin-left: 8px;
+}
+
+/* Editing the word: "2015" */
+div .word3 {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin-left: 8px;
+}
+
+/* Editing the paragraph inside de card */
+
+/* Editing the skills part insidte the card */
+
+.workContainer {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.workContainer li {
+  background: #ebebff;
+  border-radius: 8px;
+  width: 57px;
+  height: 20px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 20px;
+  color: rgba(96, 112, 255, 1);
+  padding: 4px 12px 4px 12px;
+  text-align: center;
+  margin-left: 5px;
+}
+
+/* Editing the button inside the card */
+
+.cardButton {
+  border: 1px solid rgba(96, 112, 255, 1);
+  background-color: rgba(255, 255, 255, 1);
+  width: 126px;
+  height: 48px;
+  padding: 12px;
+  margin-top: 15px;
+  border-radius: 8px;
+  color: #396df2;
+  font-weight: 500;
+  font-size: 17px;
+}
+
+/* Editing AboutMySelf Section */
+
+/* ________________________________________________________About My self__________________________________________________ */
+
+.aboutMyself {
+  padding: 114px 24px;
+  background-color: white;
+  border-top-right-radius: 6rem;
+}
+
+.aboutMyself-container {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.aboutMyself-container h4 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: rgba(23, 43, 77, 1);
+}
+
+.aboutMyself-container p {
+  font-weight: 400;
+  font-size: 16;
+  line-height: 24px;
```

Comment:
- [ ] Kindly consider to reduce the fon weight of the About myself text and align the text as the original deign to fit the figma design this will make your project more professional and presentable.
- [ ] figma
![Screenshot 2023-06-10 061505ff](https://github.com/fmalgarini/My-portfolio-Project/assets/92458236/cc967924-978c-47f5-ac00-29baf35b5d62)


- [ ] 
![Screenshot 2023-06-10 060826](https://github.com/fmalgarini/My-portfolio-Project/assets/92458236/b265b311-bc30-45cd-afc9-2af72aae9edd)